sample/SampleHost/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ConnectionStrings": {
+        "AzureWebJobsDashboard": "UseDevelopmentStorage=true",
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true"
```

Comment:
Agreed. It was just to demonstrate usage and replacement. Would it be better to put some sort of description there? Like `"AzureWebJobsDashboard": "Your connection string here",`